packages/rum/src/domain/tracing/tracer.ts

```diff
@@ -9,6 +9,7 @@ import {
 } from '@datadog/browser-core'
 
 export interface TracingContext {
+  requestIndex: number
```

Comment:
requestIndex is not specific to tracing (iirc, used initially in page activity), so I would leave it in request collection

Reply:
Do you mean something like the following?

__requestCollection.ts__
```ts
import { TracingContext, ... } from './tracing/tracer'

interface CustomContext extends TracingContext {
  requestIndex: number
}
export interface TracedFetchStartContext extends FetchStartContext, CustomContext {}
export interface TracedFetchCompleteContext extends FetchCompleteContext, CustomContext {}
export interface TracedXhrStartContext extends XhrStartContext, CustomContext {}
export interface TracedXhrCompleteContext extends XhrCompleteContext, CustomContext {}
```

__tracer.ts__
```ts
import {
  TracedFetchCompleteContext,
  TracedFetchStartContext,
  TracedXhrCompleteContext,
  TracedXhrStartContext,
} from '../requestCollection'

export interface TracingContext {
  spanId?: TraceIdentifier
  traceId?: TraceIdentifier
}
```

Reply:
What about moving these shared types to the `types.ts` file?

Reply:
Since TracingContext would then not be used in tracer, why keeping it there?